runtime/autoload/man.vim

```diff
@@ -8,6 +8,8 @@ try
   if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~# '^5'
     let s:man_find_arg = '-l'
   endif
+  call system('env MANPAGER=cat man -l ' . system('man -w man') . ' > /dev/null')
```

Comment:
I'd suggest using the `s:system` function that's defined in the file and using an array, instead of a string (as @ZyX-I already mentioned).